File: app/src/main/java/com/attendance735/attendonb/ui/vacation/pending/model/PendingResponseData.java
package com.attendance735.attendonb.ui.vacation.pending.model;

import com.attendance735.attendonb.base.commonModel.Vacation;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class PendingResponseData {

    @SerializedName("pending_vacations")
    @Expose
    public List<Vacation> pendingVacations = null;
    @SerializedName("msg")
    @Expose
    public String msg;


}

File: app/src/main/java/com/attendance735/attendonb/ui/vacation/pending/model/PendingResponseData.kt
package com.attendance735.attendonb.ui.vacation.pending.model

import com.attendance735.attendonb.base.commonModel.Vacation
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PendingResponseData {

    @SerializedName("pending_vacations")
    @Expose
    var pendingVacations: MutableList<Vacation>? = null
    @SerializedName("msg")
    @Expose
    var msg: String? = null


}
